js/plugins/chroma/package.json

```diff
@@ -47,9 +47,9 @@
   "types": "./lib/index.d.ts",
   "exports": {
     ".": {
+      "types": "./lib/index.d.ts",
```

Comment:
undo?

Reply:
This is what fixes the warnings about "will never be used because x is before y" that you see when you compile.

Reply:
the warning is wrong

Reply:
ok...